source/classes.tex

```diff
@@ -614,7 +614,7 @@
 In a \grammarterm{member-declarator} for a bit-field,
 the \grammarterm{constant-expression} is parsed as
 the longest sequence of tokens
-that could syntactically form a \grammarterm{constant-expression}.
+that would syntactically form a \grammarterm{constant-expression}.
```

Comment:
I think we can avoid the hypothetical here (and in other cases where we describe tentative parses):
```suggestion
that matches the syntax of a \grammarterm{constant-expression}.
```

Reply:
fixed